### PR TITLE
 Remove all occurrences of leap_version_at_least

### PR DESCRIPTION
**Pull request by kravciak**
Replaced by is_leap version parameter

### CODE REVIEW

**Comment by kravciak** [comment]
I would leave it to some volunteer who is in mood for it, since it's not necessary :)

Also it will be little bit trickier, since `sle_version_at_least` returns true for all non-sle products.

File: lib/main_common.pm [hunk 1]
```diff
@@ -18,7 +18,7 @@ use testapi qw(check_var get_var get_required_var set_var check_var_array diag);
 use autotest;
 use utils;
 use version_utils qw(
-  is_hyperv_in_gui is_jeos is_gnome_next is_krypton_argon is_leap is_opensuse is_sle is_sles4sap is_sles4sap_standard leap_version_at_least sle_version_at_least is_desktop_installed is_installcheck is_rescuesystem is_staging is_tumbleweed
+  is_hyperv_in_gui is_jeos is_gnome_next is_krypton_argon is_leap is_opensuse is_sle is_sles4sap is_sles4sap_standard sle_version_at_least is_desktop_installed is_installcheck is_rescuesystem is_staging is_tumbleweed
```
Comment (asmorodskyi) [inline lib/main_common.pm]: I really against such long function declaration lists , they don't give any benefits also there is high probability that they outdated so you can't actually trust them (e.g. lib/apachetest.pm). What if you just replace it with 
`use version_utils` ?

File: tests/x11/firefox/firefox_java.pm [hunk 2]
```diff
@@ -14,7 +14,7 @@
 use strict;
 use base "x11test";
 use testapi;
-use version_utils qw(leap_version_at_least sle_version_at_least);
+use version_utils qw(is_leap is_sle);
```
Comment (asmorodskyi) [inline tests/x11/firefox/firefox_java.pm]: usually you put single item just in ''

File: lib/main_common.pm (hunk 1 above)
Comment (asmorodskyi) [reply lib/main_common.pm]: ok I will do it in my PR :P

Comment (kravciak) [reply lib/main_common.pm]: This PR was not about making imports look pretty.

**Pull request merged**